### ISSUE TITLE
DEVICE MODBUS PM3255 WITH PAS600

### ISSUE THREAD

**Issue by alestoc** [Breina/PowerTagGateway#32]
Is possible integrate PM3255 by PAS600 in modbus 232/485 communication wire???

**Comment by Breina** [comment]
Yes, that's technically possible. You'd configure the PAS600 to use a Virtual Server ID for the PM3255.

I'm hesitant to implement this though. I've struggled a lot implementing devices that I don't own (#15), and this device in particular is two generations older than the Panel Servers. I've quickly went through your device's documentation `DOCA0006EN-07` and its modbus tables seem to be mostly unique, which means a lot of work to get this supported.

Any chance you are able to replace your PM3255 with a PowerTag?
